index.mjs

```diff
@@ -82,7 +82,15 @@ const loadFolder = async folder => {
 	//run all the scrapers
 	for (const { name, callback } of entries) {
 		console.log('Running scraper:', name);
-		let data = await Promise.resolve(callback());
+		let data
+		
+		try{
+			data = await Promise.resolve(callback());
+		}
+		catch{ //return errors for broken scrapers
+			data = [{updated: 0, url: 'null', created: 0, description: 'Error in scraper', name}]
+		}
+			
```

Comment:
Please try to keep PRs on topic, if you think that the smokescreen script can be improved, create an issue if you aren't sure what the problem is and need some discussion about it, or submit another PR.

Reply:
Removed